### PR TITLE
Milestone V: Crypto APP PR

### PR DESCRIPTION
**Pull request by Miliyonayalew**
# In this milestone I have
 - Set up the repository on GitHub and use Gitflow. ✔️
 - Set up React and Redux. ✔️
 - Set up testing libraries(React Testing Library and Jest). ✔️
 - Checked  there are no linter errors ✔️
 - Used the correct GitFlow ✔️
 - Documented my work in a professional way ✔️
 - Followed our list of best practices for HTML & CSS ✔️
 - Followed our list of best practices for JavaScript ✔️
 - Fetched data and update Redux store ✔️
 - Fetched data from [API ](https://documenter.getpostman.com/view/5734027/RzZ6Hzr3?version=latest) ✔️
 - Rendered a list of all Cryptocurrencies✔️
 - Render a list of details of Cryptocurrencies✔️
 - Test redux functions ✔️
 - Test components ✔️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+  ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
```
Comment (Munsa1) [inline README.md]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly do so

File: README.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  - Install all the dependencies using `npm i`
+
+To get a local copy up and running, follow these steps.
+
+### Usage
+
+To run the project, execute the following command:
+- Run `npm start`
+  
+- A new browser will open automatically by using this address [http://localhost:3000](https://stately-moonbeam-331ae3.netlify.app/#/)
```
Comment (Munsa1) [inline README.md]: - [ ] I would encourage you to remove this link as it goes to another project which is not linked to this one.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import Details from './Components/Details';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/:id" element={<Details />} />
+      </Routes>
+    </BrowserRouter>
```
Comment (Munsa1) [inline src/App.js]: - [ ] The category item detail page should have its unique route within the SPA. Kindly do so.
- Check the example below
image

File: src/Components/Details.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+            <p className="card-title">Name</p>
+            <p className="card-title">{coin.name}</p>
+          </div>
+          <li className="card-body-detail d-flex">
+            <p className="card-title">Symbol</p>
+            <p className="card-title">{coin.symbol}</p>
+          </li>
+          <div className="card-body-detail d-flex">
+            <p className="card-title">Price to USD</p>
+            <p className="card-title">
```
Comment (Munsa1) [inline src/Components/Details.js]: - [ ] Kindly follow the given design guidelines for the app.

Given design
image

Your design
image

- The following section below could be displayed like the given design

Your design
image

Given design
image

- Also see the difference between your search bar and the given design
- In the given design, the cards two per row

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Miliyonayalew, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

Your video presentation was fantastic and engaging, I enjoyed your professionalism and explanation of how you implemented all those ways of filtering through the bitcoin companies, great work.


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rustam-tolipov — APPROVED** [review]
Hi @Miliyonayalew, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**